activesupport/test/number_helper_test.rb

```diff
@@ -233,19 +233,6 @@ def test_number_number_to_human_size
         end
       end
 
-      def test_number_to_human_size_with_si_prefix
```

Comment:
I think it is better to keep the tests and use `assert_deprecated` block just to make sure we don't break the deprecated code.